kong/plugins/aws-lambda/schema.lua

```diff
@@ -20,11 +22,13 @@ return {
     },
     aws_key = {
       type = "string",
-      required = true,
     },
     aws_secret = {
       type = "string",
-      required = true,
+    },
+    use_ec2_iam_role = {
```

Comment:
Hi @thibaultcha, 
are you referring to the `use_ec2_iam_role` config property introduced in this PR? - The aws_secret and aws_key were changed to semi-optional, so whatever config was in the database is still correct after the PR (it will continue to use secrets & keys from the db config if `use_ec2_iam_role` is not set / set to false). In other words, as long as use_ec2_iam_role is not set to true the old schema is still enforced. 

Do you mean I should configure `use_ec2_iam_role` to a required property and set it to false by default through a migration script? I'm unsure of the benefits of this. Wouldn't it be better to remove the default=false flag here?